server/apps/packages/takes_package_service.py

```diff
@@ -199,8 +204,7 @@ def get_first_take_in_takes_package(self, item):
         """
         package = self.get_take_package(item)
         if package:
-            groups = package.get('groups', [])
-            refs = next((group.get('refs') for group in groups if group['id'] == 'main'), None)
+            refs = self.__get_package_refs(package)
```

Comment:
I would suggest using a single underscore here, it's already enough to mark a method as "not for public use", and the double leading underscore still doesn't make a method really private.

Double underscores with their implied name mangling are mostly meant to prevent subclasses from accidentally overriding things in a class.